streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamSlidingWindowAggregate.java

```diff
@@ -159,7 +159,7 @@ public void process(final Record<KIn, VIn> record) {
             }
 
             if (inputRecordTimestamp < windows.timeDifferenceMs()) {
-                processEarly(record.key(), record.value(), inputRecordTimestamp, closeTime);
+                processEarly(record, inputRecordTimestamp, closeTime);
```

Comment:
Seems like we don't need to pass `inputRecordTimestamp` along either. It would probably reduce confusion to omit it. Same with the other two methods.